indico/modules/events/layout/templates/layout.html

```diff
@@ -22,4 +19,58 @@
     {% call form_footer(form) %}
         <input class="i-button big highlight" type="submit" value="{% trans %}Save{% endtrans %}" data-disabled-until-change>
     {% endcall %}
+    <h2>{% trans %}Stylesheets{% endtrans %}</h2>
+    {{ form_header(css_form, classes='css-form', action=url_for('event_layout.upload_css', event)) }}
+        <div class="flashed-messages"></div>
```

Comment:
i wouldn't indent this - it's not really a block (even though i do see the point as it's inside the form)